FitNesseRoot/HttpTestSuite/FileServerTestSuite/SimpleOption/content.txt

```diff
@@ -0,0 +1,23 @@
+!-<i>OPTIONS to any existing file path except /logs should return the
+basic methods: GET,HEAD,OPTIONS,PUT,DELETE.</i>-!
+
+|script              |http browser                                              |
+|set host            |localhost                                                 |
+|set port            |5000                                                      |
+|options             |/file1                                                    |
+|ensure              |response code equals           |200                       |
+|ensure              |response header allow contains |GET,HEAD,OPTIONS,PUT,DELETE|
+|reject              |response header allow contains |POST                      |
+|options             |/no_file_here.txt                                         |
+|ensure              |response code equals           |200                       |
+|ensure              |response header allow contains |GET,HEAD,OPTIONS,PUT      |
+|reject              |response header allow contains |DELETE                    |
```

Comment:
I assume `DELETE` is excluded because the resource doesn't exist. I think it makes sense to include `DELETE` for the same reason that `GET` and `HEAD` are included. The method is allowed for that resource, even if it happens to be empty at the moment. If you `DELETE` a resource that doesn't exist, we would expect a 200 (because `DELETE` is idempotent). If the method was not allowed, we would expect a 405.

Reply:
Good point, thanks 👍
Also noticed there was no `Location` header assertion for the file-creating `PUT` so I've added that too.